styles/rentry.co/catppuccin.user.css

```diff
@@ -0,0 +1,578 @@
+          );
+          content: url("data:image/svg+xml,@{svg}");
+        }
+      }
+      .admonition-title::before,
+      &.warning > .admonition-title::before,
+      &.danger > .admonition-title::before {
+        filter: none !important;
+      }
+    }
+    // code
+    .highlighttable {
+      border-left-color: @surface0 !important;
+    }
+    .linenodiv > pre > .normal > a:not(:hover),
+    .linenodiv > pre > a:not(:hover) {
+      color: @overlay2 !important;
+    }
+    .entry-text pre,
+    .markdownx-preview pre,
+    code {
+      color: @red !important;
+    }
+    .highlight {
+      .n,
+      .nx {
+        color: @text;
+      }
+      .hll {
+        background-color: @yellow;
+      }
+      // Braces, Delimiters
+      .p {
+        color: @overlay2;
+      }
+      // Classes, Interfaces, Annotations, Metadata, Enums, Types
+      .cp {
+        color: @yellow;
+      }
+      .gd {
+        color: @red;
+      }
+      .gr {
+        color: @red;
+      }
+      .gi {
+        color: @green;
+      }
+      .go {
+        color: @overlay0;
+      }
+      .gu {
+        color: @mauve;
+      }
+      .gt {
+        color: @sapphire;
+      }
+      // Attributes
+      .na {
+        color: @yellow;
+      }
+      .no {
+        color: @yellow;
+      }
+      .ni {
+        color: @subtext0;
+      }
+      .ne {
+        color: @peach;
+      }
+      .nl {
+        color: @yellow;
+      }
+      .nn {
+        color: @text;
+      }
+      .w {
+        color: @subtext1;
+      }
+      // Escape Sequences,  Regex
+      .se {
+        color: @pink;
+      }
+      // Comments
+      .c,
+      .ch,
+      .cm,
+      .c1,
+      .cs {
+        color: @overlay2;
+      }
+      // Keyword
+      .k,
+      .kc,
+      .kd,
+      .kn,
+      .kp,
+      .kr,
+      .nt,
+      .sx,
+      .bp,
+      .kt {
+        color: @mauve;
+      }
+      // Operators
+      .o {
+        color: @sky;
+      }
+      // Constants, Numbers
+      .m,
+      .mb,
+      .mf,
+      .mh,
+      .mi,
+      .mo,
+      .il,
+      .nb {
+        color: @peach;
+      }
+      .gh,
+      .gp {
+        color: @sapphire;
+      }
+      // Strings
+      .s,
+      .sa,
+      .sb,
+      .sc,
+      .dl,
+      .s2,
+      .sh,
+      .sd,
+      .s1,
+      .cpf {
+        color: @green;
+      }
+      .nc {
+        color: @yellow;
+      }
+      // Methods, Functions
+      .nf,
+      .fm {
+        color: @blue;
+      }
+      .nd,
+      .ow {
+        color: @mauve;
+      }
+      .nv,
+      .ss,
+      .vc,
+      .vg,
+      .vi,
+      .vm {
+        color: @text;
+      }
+      .si,
+      .sr {
+        color: @red;
+      }
+      pre {
+        color: @text !important;
+      }
+      border-color: @surface0 !important;
+    }
```

Comment:
Sorry what is this again? Do we know what syntax highlighter produces this output?

Reply:
from [the previous pull request](https://github.com/catppuccin/userstyles/pull/1198)
> the syntax highlighting is using CodeMirror. and while [there is a theme for it](https://github.com/catppuccin/codemirror), it's not present in [tips-and-tricks.md](https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#how-do-i-theme-code-blocks--syntax-highlighting). also i think its [outdated](https://github.com/catppuccin/codemirror/issues/2).

Reply:
This is CodeMirror? I know the `cm-x` stuff is, the editor stuff, but the rendered markdown code blocks that this themes is different right? It's outputting completely different markup.

Reply:
maybe it's pygments? loading pygments make all the code red (not loading it keeps the code unthemed and normally highlighted)

Reply:
Ah right looks like it. Can you theme it via https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#pygments.

Reply:
that worked. i should've used it from the start but there was no signs that it uses pygments (i expected a class name)
but why are the line heights changed? and the background of the lines numbers is darker?
before:
![image](https://github.com/user-attachments/assets/2a19a31f-1494-4ef5-960b-e7c78ecb3689)
after:
![image](https://github.com/user-attachments/assets/9c3361be-1456-4ff4-931a-3c1426bf7a94)

Reply:
I guess those are styles our Pygments port sets.

Reply:
pygments broke the editor by overwriting the line height and making selecting text select the text above it (i fixed that in [`7ebadf7` (#1227)](https://github.com/catppuccin/userstyles/pull/1227/commits/7ebadf7ac021c5d35a1f046249e39d5517156732))
i don't know anything about pygments but is it necessary that the style change the line height?
![image](https://github.com/user-attachments/assets/4ba08846-5fd6-433e-8e23-e6121dedcb30)

Reply:
@thismoon line-height has been removed from the generated css 👍